features/meta-theme-color.yml

```
@@ -0,0 +1,7 @@
+name: <meta> Theme color
+description: The `<meta>` theme color is used to customize the display of the page or the surrounding browser user interface.
```

We already have features named `<link rel="modulepreload">` and `<input type="search">` so since this is also about a specific attribute value, we could call it `<meta name="theme-color">`:

```suggestion
name: '<meta name="theme-color">'
description: 'The "theme-color" `<meta>` name allows you to customize the display of the page or the surrounding browser user interface.'
```

It is pushing the limits of how syntax-y names are reasonable, but I don't know what else to suggest.

---

Will move theme color to its own PR so to not block this one.